tests/cover/test_arbitrary_data.py

```diff
@@ -49,6 +49,24 @@ def test(data):
     assert 'Draw 2: 0' in result
 
 
+def test_prints_labels_if_given_on_failure():
```

Comment:
I was going to ask if we need a test that labels *aren’t* printed when not specified, but I think that’s covered by the existing tests – since the fallback behaviour is not to print any labels, we’d notice if/when those tests broke.